collect_app/src/main/java/org/odk/collect/android/widgets/RangeWidget.java

```diff
@@ -203,6 +217,17 @@ public boolean onTouch(View v, MotionEvent event) {
         });
     }
 
+    // check if it's RTL languages
+    public static boolean isRTL() {
+        return isRTL(Locale.getDefault());
+    }
+
+    public static boolean isRTL(Locale locale) {
```

Comment:
Both those methods already exist in `QuestionWidget` so please use them.

Reply:
Thanks for remaining !
I will update my submits ;)
Ps, I think PR#2048 also need review if you have spare time.
Have a good day !